### DISCUSSION TITLE
Names and BMS warning messages

### DISCUSSION THREAD

**Discussion by Gr33n93** [Louisvdw/dbus-serialbattery#700] [Q&A] [answered]
The name of the BMS Still does not persist after a restart. Is this function still planned?

After the driver update from 0.14.03 to 1.0.20230531 my Daly BMS now show warning messages when the SOC falls below 20% in the BMS, that was not so before. I assume that warning messages simply did not work before.

However, I am looking for a solution to this problem. One idea was a conversion like the Smartshunt from Victron, but I have noticed that one of my 3 Daly BMS as soon as I send no SOC value only restarts this makes it but not always but only in 50% of cases. The firmware is the same on all BMS.

Can I somehow disable certain warnings in the driver?

**Comment by mr-manuel** [comment, marked as answer]
> The name of the BMS Still does not persist after a restart. Is this function still planned?

See https://github.com/Louisvdw/dbus-serialbattery/issues/100.

> After the driver update from 0.14.03 to 1.0.20230531 my Daly BMS now show warning messages when the SOC falls below 20% in the BMS, that was not so before. I assume that warning messages simply did not work before.

Correct. The warning messages did not work before. 

> However, I am looking for a solution to this problem. One idea was a conversion like the Smartshunt from Victron, but I have noticed that one of my 3 Daly BMS as soon as I send no SOC value only restarts this makes it but not always but only in 50% of cases. The firmware is the same on all BMS.
> 
> Can I somehow disable certain warnings in the driver?

They are triggered directly from the Daly BMS, so you have to change the settings/thresholds there.

**Reply by Gr33n93** [reply]
Thanks for the feedback.

Note: Daly only allows you to set the alarm values with the PC software and not with the smartphone app. 

for the name of the Daly BMS or even the date of commissioning is the same, only the PC software goes.

in my case the software is called TOPBMS-V 1.17

maybe you can include this info under special feature of the BMS.

**Reply by mr-manuel** [reply]
There are news for the name persistance. Check the link I posted above.

Can you post some screenshots of the software, so I can add this?
